extensions/amp-inputmask/README.md

```diff
@@ -0,0 +1,117 @@
+<!--
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# \`amp-inputmask\`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Provides input masking capabilities to inputs in AMP forms</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-inputmask" src="https://cdn.ampproject.org/v0/amp-inputmask-0.1.js">&lt;/script></code></td>
+  </tr>
+  <!-- TODO(@cvializ) -->
+  <!-- <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>FILL THIS IN</td>
+  </tr> -->
+</table>
+
+## Behavior
+
+`amp-inputmask` enables the `mask` and `mask-output` attributes on `input` elements. These allow document authors to specify input masks for their form elements.
```

Comment:
/cc @CrystalFaith for doc review.

Reply:
The language looks really good! I see there is a todo - but its hard for me to sign off without a code sample. Its unclear to me how this is applied to an input(or replaces the input element?) - and then how I'm able to specify the mask attributes.